seed/static/seed/js/controllers/column_settings_controller.js

```diff
@@ -66,6 +67,14 @@ angular.module('BE.seed.controller.column_settings', [])
         {id: 'geometry', label: $translate.instant('Geometry')}
       ];
 
+      $scope.changeText = function(btnText) {
+        console.log(btnText);
```

Comment:
Haven't reviewed fully, but noticed this console.log

Reply:
whoops, thank you!